.github/workflows/ci.yml

```diff
@@ -14,23 +14,22 @@ jobs:
     timeout-minutes: 5
 
     steps:
-      - uses: actions/setup-go@v5
+      - uses: actions/setup-go@3041bf56c941b39c61721a86cd11f3bb1338122a # v5.2.0
         with:
           go-version: 1.21.x
 
       - name: Set env
         shell: bash
         # TODO(thaJeztah): remove working-directory, path, and GOPATH once project-checks stops needing them; see https://github.com/containerd/nri/pull/53/commits/872fb0ce3dce136f3ae67c068ce78607565194ef#r1324366346
         run: |
-          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
```

Comment:
Do we still need GOPATH?

Reply:
It can be removed. I think it was a miss from the original PR/commit. 

Ref: https://github.com/containerd/nri/commit/872fb0ce3dce136f3ae67c068ce78607565194ef and https://github.com/containerd/nri/pull/53/commits/e33ac3e43394ead24dbb4a407dd76c00c5780737